stories/grid.stories.tsx

```diff
@@ -29,6 +29,27 @@ export function GridWellPlate() {
       rows={8}
       columns={12}
       displayAsGrid
+      headerStyle={({ position, label }) => {
+        if (label === '') {
+          return { backgroundColor: 'white' };
+        }
+
+        if (
+          position.column % 2 === 0 &&
+          position.row === 0 &&
+          !isNaN(Number(label))
```

Comment:
I think you meant `Number.isNaN` because this one will return `true` also if the passed value is the NaN special value.

But I think you can skip this check anyway because it is handled above. (I assume the check is also only valid for the upper left corner).